csi/controller_server.go

```diff
@@ -32,8 +32,10 @@ const (
 	tickBackupInitiation    = 5 * time.Second
 	backupStateCompleted    = "Completed"
 
-	csiSnapshotTypeLonghornSnapshot = "ss"
-	csiSnapshotTypeLonghornBackup   = "bs"
+	csiSnapshotTypeLonghornSnapshotLongName = "snapshot"
+	csiSnapshotTypeLonghornBackupLongName   = "backup"
+	csiSnapshotTypeLonghornSnapshot         = "ss"
+	csiSnapshotTypeLonghornBackup           = "bs"
```

Comment:
Sorry. I was thinking we will use `snap/bak` in both CSI snapshot and Longhorn volume... 
Using 2 kinds of expression is confusing.

Reply:
We already use `bs` in the csi snapshot: https://longhorn.io/docs/1.2.3/snapshots-and-backups/csi-snapshot-support/restore-a-backup-via-csi/#restore-a-backup-that-has-no-associated-volumesnapshot

Reply:
But the current `bs` is for `VolumeSnapshotContent`.

For `VolumeSnapshotClass`, I realized we should change from `bs/ss` to `snap/bak` or `snapshot/backup`.

Reply:
We can add backward compatible code for the previous implementation. e.g.,
```go
 if csiSnapshotType == "" || csiSnapshotType == "bs" || csiSnapshotType == csiSnapshotTypeLonghornBackupLongName {
	// For backward compatibility, type "" or "bs" is considered as csiSnapshotTypeLonghornBackupLongName
	rsp, err = cs.createCSISnapshotTypeLonghornBackup(req)
}
```

Reply:
it's better to have a function/method wrapper for the backward compatibility condition check instead of an inline condition

Reply:
After discussing with the team, it seems that there were some misunderstandings about how we would agree on this fix.  Therefore, I have prepare 2 PRs for the sake of clarity:
* The first approach is this PR
* The second approach is the PR https://github.com/longhorn/longhorn-manager/pull/1240

Please let me know what you think? @joshimoo @shuo-wu @jenting @innobead